server/request_handler.go

```diff
@@ -38,7 +40,7 @@ func (r *RpcRequestHandler) process() {
 	// At end of request, log the time it needed
 	defer func() {
 		timeRequestNeeded := time.Since(r.timeStarted)
-		r.logger.log("request took %.6f sec", timeRequestNeeded.Seconds())
+		log.Info("[process] Time taken to process request", "request took", fmt.Sprintf("%.6f sec", timeRequestNeeded.Seconds()))
```

Comment:
```suggestion
		log.Info("[process] Time taken to process request", "sec", timeRequestNeeded.Seconds())
```

Better to keep the values unformatted, and to have only a single word as key

Reply:
Updated as  suggested